store/store_options.go

```diff
@@ -7,27 +7,18 @@ import (
 type Parameters struct {
 	// RecentBlocksCacheSize is the size of the cache for recent blocks.
 	RecentBlocksCacheSize int
-
-	// AvailabilityCacheSize is the size of the cache for accessors requested for serving availability
-	// samples.
-	AvailabilityCacheSize int
```

Comment:
Another concern that I had, but forgot to bring up during our sync is this. Where do you see this configuration param moving with the approach in this PR.
IMO these cache configurations should be close together, as they were.

Reply:
I guess we can keep it as it is and then use it nodebuilder where we construct `WithCache`

Reply:
Yes, it will be set externally.